src/etc/man/cargo-metadata.1

```diff
@@ -226,7 +233,7 @@ The output has the following format:
                 */
                 "deps": [
                     {
-                        /* The name of the dependency.
+                        /* The name of the dependency\(aqs library target.
```

Comment:
Weird.  I guess apostrophes need escaping here, and this is how it's done?

Reply:
Yea, it's a troff "apostrophe-quote" character.